sqlalchemy-stubs/sql/operators.pyi

```diff
@@ -46,7 +46,7 @@ class ColumnOperators(Operators):
     def notilike(self, other, escape: Optional[Any] = ...): ...
     def is_(self, other): ...
     def isnot(self, other): ...
-    def startswith(self, other, **kwargs): ...
+    def startswith(self, other: str, **kwargs): ...
```

Comment:
Might be useful to add `other: str` to the following string methods in this PR?

Reply:
It is better to make a separate PR, I will merge this one now.